app/router.js

```diff
@@ -20,6 +21,14 @@ module.exports.bind = app => {
   app.use(directDebitGuarantee.router)
   app.use(confirmation.router)
   app.use(changePaymentMethod.router)
+  app.get('/sentry-test-default', (req, res) => {
```

Comment:
If these are for integration tests that are just run for this app they could probably be added in the test runner itself. 

As this module exports the Express app you can add handlers as needed for different environments. 

If they're for end to end/ external tests then there's no real problem with having them around.

Reply:
Yeah they're just very temporary to make it easy to trigger and error whilst we roll it out through the environments and make sure it's all working. We're going to remove them in the next PR.